### PR TITLE
[STACK wlutz-> config -> base] Wlutz experimental work

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

File: lib/pymedphys/_experimental/streamlit/apps/wlutz/_utilities.py [hunk 1]
```diff
@@ -22,6 +23,17 @@
...
 
 
+def filter_by(dataframe, column, value):
+    filtered = dataframe.loc[dataframe[column] == value]
+
+    return filtered
+
+
+# https://stackoverflow.com/a/16090640/3912576
+def natural_sort_key(s, _nsre=re.compile("([0-9]+)")):
```
Comment (sjswerdloff) [inline lib/pymedphys/_experimental/streamlit/apps/wlutz/_utilities.py:33]: Consider import natsort instead of coding locally

**Review by sjswerdloff — APPROVED** [review]
Strip body of code that is completely commented when moving out of experimental.
More parameter typing would be nice

File: lib/pymedphys/_experimental/streamlit/apps/wlutz/_utilities.py (hunk 1 above)
Comment (SimonBiggs) [reply lib/pymedphys/_experimental/streamlit/apps/wlutz/_utilities.py:33]: Yup, good call. I'll do that before merge.

**Comment by SimonBiggs** [comment]
> Strip body of code that is completely commented when moving out of experimental.

Yup, certainly 🙂 

> More parameter typing would be nice

I'll keep that in mind and see if I can make sure that happens as I am refactoring.

**Pull request merged**